devices/legrand.js

```diff
@@ -320,4 +320,62 @@ module.exports = [
+            exposes.binary('led_when_on', ea.STATE_SET, 'ON', 'OFF').withDescription('Enables the LED when the light is on')],
+        meta: {multiEndpoint: true},
+        endpoint: (device) => {
+            return {left: 1, right: 2};
+        },
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genIdentify', 'genBinaryInput', 'genOnOff',
+                'manuSpecificLegrandDevices']);
+            await reporting.onOff(endpoint);
+            const endpoint2 = device.getEndpoint(2);
+            await reporting.bind(endpoint2, coordinatorEndpoint, ['genIdentify', 'genBinaryInput', 'genOnOff']);
+            await reporting.onOff(endpoint2);
+        },
+        onEvent: async (type, data, device) => {
+            await legrand.pairing_security_event(type, data, device);
+        },
+    },
+    {
+        zigbeeModel: [' Dimmer switch with neutral\u0000\u0000\u0000\u0000'],
+        model: 'LEG600090',
+        vendor: 'Legrand',
+        description: 'Wired dimmable switch with neutral LEG600081',
+
+        extend: extend.light_onoff_brightness({noConfigure: true}),
+        fromZigbee: [fz.brightness, fz.identify, fz.on_off, fz.lighting_ballast_configuration],
+        toZigbee: [tz.light_onoff_brightness, tz.legrand_settingAlwaysEnableLed, tz.legrand_settingEnableLedIfOn,
+            tz.legrand_settingEnableDimmer, tz.legrand_identify, tz.ballast_config],
+        exposes: [e.light_brightness(),
+            exposes.numeric('ballast_minimum_level', ea.ALL).withValueMin(1).withValueMax(254)
+                .withDescription('Specifies the minimum brightness value'),
+            exposes.numeric('ballast_maximum_level', ea.ALL).withValueMin(1).withValueMax(254)
+                .withDescription('Specifies the maximum brightness value'),
+            exposes.binary('dimmer_enabled', ea.STATE_SET, 'ON', 'OFF').withDescription('Allow the device to change brightness'),
+            exposes.binary('permanent_led', ea.STATE_SET, 'ON', 'OFF').withDescription('Enable or disable the permanent blue LED'),
+            exposes.binary('led_when_on', ea.STATE_SET, 'ON', 'OFF').withDescription('Enables the LED when the light is on')],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            await extend.light_onoff_brightness().configure(device, coordinatorEndpoint, logger);
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genIdentify', 'genOnOff', 'genLevelCtrl', 'genBinaryInput',
+                'manuSpecificLegrandDevices', 'lightingBallastCfg']);
+            await reporting.onOff(endpoint);
+            await reporting.brightness(endpoint);
+        },
+        onEvent: async (type, data, device) => {
+            await legrand.pairing_security_event(type, data, device);
```

Comment:
`pairing_security_event` this method does not exist, I'm not sure why this was added.

Reply:
oopss yes was part of a try to support direct greenpower association but doesn't work and I didn't clean correctly my code.
I review my code and fix it.
I'm sorry for that.